library/init/meta/simp_tactic.lean

```diff
@@ -43,9 +43,9 @@ If your lemma is not being added, you can see the reasons by setting `set_option
 - `LHS` should not occur within a hypothesis `hᵢ`.
 
  -/
-meta constant simp_lemmas.add : simp_lemmas → expr → tactic simp_lemmas
```

Comment:
These new extra argument is probably going to break lots of stuff in mathlib.  Can you maybe at least add a default for the new argument?